py/picca/tests/delta_extraction/abstract_test.py

```diff
@@ -175,20 +178,28 @@ def compare_fits(self, orig_file, new_file):
                             print(f"Column {col} in HDU {orig_header['EXTNAME']} "
                                   "missing in new file")
                         self.assertTrue(col in new_data.dtype.names)
+                        
+                        if 'IVAR' in new_data.dtype.names:
```

Comment:
why is this necessary? Even if masked pixels are kept, why would they be different in different runs of the tests?

Reply:
This applies to the rest of the changes in this file from here onwards